proposals/low-level-struct-improvements.md

```diff
@@ -410,6 +410,86 @@ Detailed Notes:
     - The core library contains the feature flag indicating support for `ref` fields
     - The `langversion` value is 11 or higher
 
+### Syntax
+[12.6.2 Local variable declarations](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/statements.md#1262-local-variable-declarations): added `local_variable_scope_modifier`.
+```antlr
+local_variable_declaration
+    : local_variable_scope_modifier? local_variable_type local_variable_declarators
+    ;
+
+local_variable_scope_modifier
+    : local_variable_mode_modifier
+    | 'scoped' local_variable_mode_modifier?
+    ;
+
+local_variable_mode_modifier
+    : 'ref' 'readonly'?
+    ;
+```
+
+[12.9.4 The `for` statement](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/statements.md#1294-the-for-statement): added `local_variable_scope_modifier` _indirectly_ from `local_variable_declaration`.
+
+[12.9.5 The `foreach` statement](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/statements.md#1295-the-foreach-statement): added `local_variable_scope_modifier`.
+```antlr
+foreach_statement
+    : 'foreach' '(' local_variable_scope_modifier? local_variable_type identifier 'in' expression ')'
+      embedded_statement
+    ;
+```
+
+[11.6.2 Argument lists](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#1162-argument-lists): added `scoped` for `out` declaration variable.
+```antlr
+argument_value
+    : expression
+    | 'in' variable_reference
+    | 'ref' variable_reference
+    | 'scoped'? 'out' local_variable_type? identifier
+    ;
+```
+
+[14.6.2 Method parameters](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/classes.md#1462-method-parameters): added `parameter_scope_modifier`.
+```antlr
+fixed_parameter
+    : attributes? parameter_modifier? type identifier default_argument?
+    ;
+
+parameter_modifier
+    : parameter_scope_modifier
+    | 'this' parameter_scope_modifier?
+    ;
+
+parameter_scope_modifier
```

Comment:
Similar question here. I think the organization makes sense, but I'm not sure about the name.